### DISCUSSION TITLE
V2

### DISCUSSION THREAD

**Discussion by hgraphql** [hashgraph/hedera-wallet-connect#150] [Ideas]
This library follows semantic versioning - https://semver.org/.

> MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes

This discussion topic is meant to discuss features / changes that require a breaking change.

All feature additions can still be added to v 1.X.X releases. One example of a breaking change for the next version is https://github.com/hashgraph/hedera-wallet-connect/issues/53 because we will remove deprecated functionality.

**Comment by MrBns** [comment]
this Discussion Opened on May 30,2024.  a long time has been passed. what what you guys are decided. ??!

still going with deprecated wallet-connect lib. like `qrcode-model` `qrcode-core` etc. 

a lots of thing is going to deprecated.. please stop working on deprecated thing. in wallet space Hedera needs serious update.
